usagov/usagov-2021
# USAGOV-1435-feedback-focus-outline-update: add comment to override in…

…line style


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1435

## Description

Add "!important" to css to override inline style
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Open the _uswds-theme-custom-styles.scss file and goto line 703 to verify the padding has "!important" added to it. The outline for the button can only be verified on production.


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
